perception/radar_fusion_to_detected_object/src/radar_object_fusion_to_detected_object_node/radar_object_fusion_to_detected_object_node.cpp

```diff
@@ -1,5 +1,5 @@
 
-// Copyright 2022 TIER IV, Inc.
+// Copyright 2023 TIER IV, Inc.
```

Comment:
Please don't overwrite the original copyright year.

https://autowarefoundation.github.io/autoware-documentation/main/contributing/license/ indicates that this file follows the format from https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/blob/87c5e5880a18068116dd886ad56e1bfc29e694c4/CONTRIBUTING.md so you can update to `Copyright 2022-2023 [your company], TIER IV, Inc.` or you can leave it as is.

Reply:
Got it.